Doc/library/dataclasses.rst

```diff
@@ -536,8 +536,8 @@ Mutable default values
 
      o1 = C()
      o2 = C()
-     o1.add(1)
-     o2.add(2)
```

Comment:
I'd rather keep this interface, and instead make the change in the `add()` method. Especially so since the name of the parameter is `element`.  `add()` would become `self.x.append(element)`.